messages/de.json

```diff
@@ -0,0 +1,1393 @@
+{
+    "_metadata": {
+        "*": "For assistance in configuring this metadata, please see https://werewolf.chat/Translation#Metadata",
+        "fallback": null,
```

Comment:
should be "fallback": "en" so that English messages are used in case they're missing from the German file. We will be modifying the English file from time to time and adding new things to it, and missing a fallback means anyone running the bot in German will crash instead of just getting some potentially untranslated messages.